xskillscore/core/np_deterministic.py

```diff
@@ -139,8 +139,7 @@ def _effective_sample_size(a, b, axis, skipna):
 
 
 def _pearson_r(a, b, weights, axis, skipna):
-    """
-    ndarray implementation of scipy.stats.pearsonr.
+    """ndarray implementation of scipy.stats.pearsonr.
```

Comment:
Is this a pre-commit thing or a user thing? I've seen it done both ways. I prefer the way it is done before. I know it adds an extra line but I naturally read `ndarray implementation` better as my brain doesn't like there not being a space (or line) between `"""` and `ndarray`

Reply:
I did this manually. It's the proper way to do docstrings in numpy and google format. The idea is that the description should be short enough to fit on one line there. (https://numpydoc.readthedocs.io/en/latest/format.html). I'm happy to revert it back. I don't have a strong preference.